### DISCUSSION TITLE
doris on es create table successs，but  select failed to connect es server port 80 refuesd

### DISCUSSION THREAD

**Discussion by lvweiwei007** [apache/doris#7092] [A - General / Q&A]
CREATE EXTERNAL TABLE `test_es1` (

gctp2 varchar(500),

gctp3 varchar(500),

lhy_zt varchar(30),

gctp1 varchar(500),

hpzl varchar(10),

lhy_clpp2 varchar(20),

lhy_clpp1 varchar(20),

bmmc varchar(50),

sbmc varchar(50),

sjly bigint,

gcbh varchar(20),

hpys varchar(10),

scsj datetime,

lhy_syxz varchar(20) ,

cllx varchar(10),

id varchar(50),

csys varchar(10),

cdh smallint,

sj_lhy varchar(20),

jrsj datetime,

wfdm varchar(10),

dlmc varchar(200),

rkhs  bigint,

dwbh  varchar(50),

ffsj datetime,

hjhs bigint,

hphm varchar(10),

dldm varchar(10),

glbm varchar(20),

dwmc varchar(500),

lhy_qzbfqz varchar(20),

sbxh varchar(20),

sd float,

clsd float,

lhy_cllx varchar(20),

xsz int,

sbbh varchar(20),

cwkc float,

jrhs bigint,

ffhs bigint,

jgsj datetime,

lhy_syr varchar(20),

rksj datetime,

lhy_csys varchar(20),

hjsj datetime,

fxlx  varchar(10),

lhy_sjhm  varchar(50),

lhy_xzqh  varchar(20),

lhy_clxh  varchar(50),

lhy_state_lhy  varchar(20),

csb int,

lhy_sfzmhm  varchar(20),

lhy_yxqz  varchar(20),

state_lhy varchar(50)

) ENGINE=ELASTICSEARCH

PROPERTIES (

"hosts" = "http://es1.zyjj:9200",

"index" = "veh_pass202110",

"type" = "doc",

"user" = "elastic",

"password" = "ela123456"

);
![Uploading 1636599117.png…]()

**Comment by lvweiwei007** [comment]
![1636599117](https://user-images.githubusercontent.com/94092447/141391847-c89a7756-8487-42d5-a532-dc0c01cf4560.png)

**Comment by ChPi** [comment]
用的什么版本。看看fe日志的报错，url是什么

**Reply by lvweiwei007** [reply]
version	5.1.0
version_comment	Doris version 0.14.0-release-Unknown

fe日志
![1636696723(1)](https://user-images.githubusercontent.com/94092447/141418026-677d65f0-cf9a-4e72-98b3-3d2e843fb0a3.jpg)
：

url啥意思啊 没找到

**Reply by lvweiwei007** [reply]
be日志：
![1636697647(1)](https://user-images.githubusercontent.com/94092447/141419781-8cb7d447-a8c9-4f75-96c7-6accd5a547a9.jpg)

**Reply by ChPi** [reply]
再看看对应be的日志。我之前遇到过类似的，是es配置导致的，不过这个修复了

**Reply by lvweiwei007** [reply]
感觉像是es配置 ，但是网络 和9200端口又是通的的；be日志 我刚传了图呢 你帮看看，问题卡了几天了..

**Reply by lvweiwei007** [reply]
问题已解决

**Comment by 18810177021** [comment]
哈喽，您好，请教个问题，执行mysql -uroot -h127.0.0.1 -P 8030报ERROR 2013 (HY000): Lost connection to MySQL server at 'reading initial communication packet', system error: 0，fe 与mysqlclient 在同一台服务器，抓取过包，没有问题

**Reply by dataalive** [reply]
> 哈喽，您好，请教个问题，执行mysql -uroot -h127.0.0.1 -P 8030报ERROR 2013 (HY000): Lost connection to MySQL server at 'reading initial communication packet', system error: 0，fe 与mysqlclient 在同一台服务器，抓取过包，没有问题

这个大概率是因为你编译的doris的jdk版本高于运行doris的jdk版本。比如在docker中用jdk11编译，而生产中却是jdk8

**Comment by hf200012** [comment]
PROPERTIES  add 
nodes_discovery: "false"
